### PR TITLE
Add Fra146's hackbar

### PR DESCRIPTION
**Pull request by Fra146**

### CODE REVIEW

**Comment by Fra146** [comment]
Please add "hacktoberfest-accepted" when merging

**Comment by tasnimzotder** [comment]
@Fra146 Sure

File: _hackbars/fra146.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+layout: hackbar
+title: Manifestation
+author: Fra146
+---
+
+Opensource is a beautiful thing, cause he include everyone in the creation of something
+
+---
+
+Climate change is breaking earth! Let's move!
```
Comment (tasnimzotder) [inline _hackbars/fra146.md]: hey @Fra146, can you change the sentence - 
```suggestion
## Climate change is breaking earth! Let's move!
```

**Comment by tasnimzotder** [comment]
Thanks @Fra146 for contributing :)

**Pull request merged**